Assets/css/style.css

```diff
@@ -286,6 +365,34 @@ textarea {
   justify-content: space-between;
 }
 
+#mobile_menu {
```

Comment:
- [ ] [optional] Your menu has some differences in the navigation links ```margin``` and the ```hr``` thickness, you can try to modify them so your design can be as close as possible to the Figma one

![Screenshot_20220510_190024](https://user-images.githubusercontent.com/25227022/167692992-206e0350-6a21-48f6-9295-5ffbb792fa9f.png)